lib/moon/components/table.ex

```diff
@@ -31,12 +35,20 @@ defmodule Moon.Components.Table do
         <Paging paging_info={@paging_info} paging_click={@paging_click} limit={@limit} offset={@offset} />
       {/if}
       <div class="w-full overflow-x-scroll">
-        <table class="text-sm border-collapse border-t border-beerus" style="min-width: 100%;">
+        <table
+          class={"text-sm border-separate border-spacing-x-0 border-spacing-y-#{@row_gap} border-beerus"}
```

Comment:
I think we should just have arbitary class, where additional classes can be passed... also we could have 

`prop(border_spacing, :css_class, default: "border-spacing-y-1")`